package.json

```diff
@@ -26,11 +26,17 @@
   "devDependencies": {
     "@automattic/custom-colors-loader": "automattic/custom-colors-loader",
     "@automattic/dops-components": "automattic/dops-components",
-    "babel-core": "5.8.25",
+    "babel-core": "^6.8.0",
```

Comment:
Thanks. Yes, it was intended but maybe not well thought. Thanks for the recommendation.